app/src/main/java/com/keystone/cold/ui/fragment/main/web3/EthTxConfirmFragment.java

```diff
@@ -205,14 +206,26 @@ private void updateNetworkName() {
     private void processAndUpdateTo() {
         AppExecutors.getInstance().diskIO().execute(() -> {
             String to = txEntity.getTo();
+            String ens = viewModel.ensAddress(to);
```

Comment:
function name should describe what this function do actually. Do not use a noun to naming an action.